src/components/templates/Search/sort.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
```

Comment:
there is a lot going on in here but if I see this correctly it tries to reorder the existing results on screen. This will not work because we only show 1 page of the search results and when resorting we want to resort across all pages, not just the one displayed. So changing the sorting needs to trigger a complete new search